nanshe_workflow/__init__.py

```diff
@@ -4,3 +4,9 @@
 from ._version import get_versions
 __version__ = get_versions()['version']
 del get_versions
+
+import os
+os.environ["DASK_CONFIG"] = os.path.join(
+    os.path.dirname(os.path.abspath(__file__)), "dask_config.yaml"
+)
+del os
```

Comment:
Ensures that the Dask Distributed configuration points to the one included in the workflow.